lib/shortcuts/glue-orc-table.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const GlueTable = require('./glue-table');
+
+class GlueOrcTable extends GlueTable {
+  constructor(options = {}) {
+    let { Parameters } = options;
+    const { Location } = options;
+
+    const required = [Location];
+    if (required.some((variable) => !variable))
+      throw new Error('You must provide a Location');
```

Comment:
stub leftovers?

Reply:
Nope, this is supposed to be here. I think that's what you're asking?

Reply:
yep 😄 In that case why is Location required?

Reply:
Can a table with these input/output formats work as intended without a Location?

Reply:
Yes, location denotes the default path where partitions will be stored, but you're free to explicitly register a partition into a different path, even if Location is defined.

Reply:
Okay I think I see the behavior. Glue only needs to know where each partition's files are located, and the table-level specification just provides a default. 

If you specify a table-level Location but you don't specify a partition-level Location, where does Glue expect the partition data to reside? For example say my table's Location is `s3://bucket/my/table` and I write a partition to it for `partion_column=value`. Where would I put the ORC data on S3 in order for Glue to find it?

Do you see any benefit to forcing table-level declaration of the location? This makes for more useful metadata.

![image](https://user-images.githubusercontent.com/515424/66004129-c2edcf80-e45c-11e9-9667-770c48364ea4.png)

Too opinionated?

Reply:
Another thing, glue won't automatically pick up partitions that are written to the correct default location, you still need to either tell it to register that particular partition or to "repair" and pick up missing partitions. So using Location won't save an sdk call.

`${LocationUri}/${Location}/dt={}/hr={}` would be the location for partitions expected by a table with dt and hr partitions.

Im familiar enough with the value of `Location` to want to make it required. I can see some confusion coming from it not being prescriptive, you could end up with `location: "bogus"` in the params and still have a functioning table.

Reply:
> Do you see any benefit to forcing table-level declaration of the location? This makes for more useful metadata that's viewable in the Glue UI.

A lot of these parameters are Hive parameters leaking through the Glue abstraction. This is one of them. `Location` is not required by Hive, so I don't think we should make it required.